### DISCUSSION TITLE
정말 오랜만에보는 숫자들

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#41] [Show and tell]
![image](https://user-images.githubusercontent.com/62102830/194746138-e99729e5-035a-4429-a612-80cddbeb8fa3.png)

10만개의 데이터를 넣을때 이정도 걸렷엇군요

![image](https://user-images.githubusercontent.com/62102830/194746186-2d265c4c-c24a-4ea1-b80f-24828da0a22b.png)

신기하게 100만개 넣으니까 딱 10배 되네요 ㅋㅋ

**Comment by Woon-2** [comment]
더욱 더 데이터가 많아지면 `page fault`나 `cache miss`가 발생해 기하급수적으로 느려질 겁니다.
`linked list`의 한계지요.
